backend/models.py

```diff
@@ -1,3 +1,66 @@
-# from django.db import models
+from django.db import models
 
 # Create your models here.
+
+
+class Faculty(models.Model):
+    name = models.CharField(max_length=150)
+
+
+class FieldOfStudy(models.Model):
+    faculty = models.ForeignKey(Faculty, on_delete=models.CASCADE)
+    name = models.CharField(max_length=150)
+    degree = models.IntegerField()
+
+
+class Yearbook(models.Model):
+    field_of_study = models.ForeignKey(FieldOfStudy, on_delete=models.CASCADE)
+    begin = models.DateField()
+    end = models.DateField()
+    available_capacity = models.IntegerField()
+
+
+class Student(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    date_of_birth = models.DateField()
+    gender = models.CharField(max_length=50)
+    year_of_exam = models.IntegerField()
+    city = models.CharField(max_length=80)
+
+
+class Graduate(models.Model):
```

Comment:
To jest informacja o tym, że ktoś ukończył studia? Nie wiem czy będziemy mieli takie dane. Raczej bym informacje o tym, że ktoś jest absolwentem dodał do tabeli kandydatów. A nawet być może jakoś nawet bardziej rozdzielił apliakcje na pierwszy stopień a na drugi

Reply:
Problem z tym, że ktoś jest absolwentem jest taki, że to może być wiele kierunków, a robienie list raczej nie działa w SQLu. Dlatego właśnie jest nowa tabelka, w której mamy zapisane, kto co skończył, z jakim wynikiem

Reply:
hmm wydaje mi się, że jak się składa na studia magisterskie to i tak liczy się jeden kierunek pierwszego stopnia i nawet jak masz dwa skończone to musisz wybrać jeden, który podajesz i on się liczy, więc nie ma problemu z listami. Ale to też jest chyba do uzgodnienia z klientem